services/scheduler/job.go

```diff
@@ -0,0 +1,34 @@
+package scheduler
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/percona/pmm-managed/models"
+)
+
+type Job interface {
+	Do(ctx context.Context) error
+	Type() models.ScheduleJobType
+	Data() models.ScheduleJobData
+}
+
+type EchoJob struct {
+	jobsDeps
+	models.EchoJobData
```

Comment:
Can we have simpe `message string` here and convert `EchoJob` to the model right before persisting? IMHO it will be more explicit and simpler.